### PR TITLE
Make the contribution popover default to hidden to prevent it from blinking

### PR DESCRIPTION
**Pull request by joshJarr**
Stops the contribution banner showing then hiding (blinking) after being dismissed.
Default the banner to close, open it if there is no LocalStorage state.

This also solves the issue of hiding the banner when JS is disabled, so I removed that check.

### CODE REVIEW

File: kuma/static/js/contribution-handler.js [hunk 1]
```diff
@@ -153,20 +153,28 @@
...
+                // Show if LS does not exist
+                showPopover();
             }
+        } else {
+            // Show if LS does not exist
+            showPopover();
         }
     }
 
+    // Removes 'hidden' class from popover
```
Comment (matthewhall) [inline kuma/static/js/contribution-handler.js]: Doesn't just from the hidden class though, it also adjusts the `aria-` attribute. Can we make this a proper JSDoc comment as well please?

File: kuma/static/js/contribution-handler.js [hunk 2]
```diff
@@ -153,20 +153,28 @@
...
-                    if (disabledStorageItem.timestamp + CONTRIBUTIONS_DISABLED_EXPIRATION > date) {
-                        // Keep the popover hidden if we aren't passed the expiration date yet.
-                        popoverBanner.addClass('hidden');
-                        popoverBanner.attr('aria-hidden', true);
-                    } else {
-                        // Remove the item if it has expired.
-                        localStorage.removeItem('contributionsPopoverDisabled');
-                    }
+                if (disabledStorageItem.value 
+                    && disabledStorageItem.timestamp + CONTRIBUTIONS_DISABLED_EXPIRATION < date) {
```
Comment (matthewhall) [inline kuma/static/js/contribution-handler.js]: Please indent by one tab more.

File: kuma/contributions/jinja2/contributions/includes/contribution-form.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div id="contribution-popover-container" class="contribution-banner {% if is_popover %}contribution-popover is-collapsed{% endif %}"{% if is_popover %} aria-expanded="false" aria-hidden="false"{% endif %}>
+<div id="contribution-popover-container" class="contribution-banner {% if is_popover %}contribution-popover is-collapsed hidden{% endif %}"{% if is_popover %} aria-expanded="false" aria-hidden="true"{% endif %}>
```
Comment (matthewhall) [inline kuma/contributions/jinja2/contributions/includes/contribution-form.html]: Nicer to use `is-hidden`?

Comment (joshJarr) [reply kuma/contributions/jinja2/contributions/includes/contribution-form.html]: hidden is their class, but I can add an alias for this

File: kuma/static/js/contribution-handler.js (hunk 2 above)
Comment (joshJarr) [reply kuma/static/js/contribution-handler.js]: but it's on the correct line? 
```
function checkPopoverDisabled() {
        if (win.mdn.features.localStorage) {
            var disabledStorageItem = localStorage.getItem('contributionsPopoverDisabled');
            var date = new Date().getTime();

            if (disabledStorageItem) {
                // Parses the stringified storage item
                disabledStorageItem = JSON.parse(disabledStorageItem);

                if (disabledStorageItem.value 
                    && disabledStorageItem.timestamp + CONTRIBUTIONS_DISABLED_EXPIRATION < date) {
                    // Remove the item if it has expired.
                    localStorage.removeItem('contributionsPopoverDisabled');
                    showPopover();
                }
            } else {
                // Show if LS does not exist
                showPopover();
            }
        } else {
            // Show if LS does not exist
            showPopover();
        }
    }

```

Comment (joshJarr) [reply kuma/static/js/contribution-handler.js]: ohh I see! My bad so the ` && disabledStorageItem.timestamp + CONTRIBUTIONS_DISABLED_EXPIRATION < date) {` should be unindented by 1 tab, or the contents of that block should be indented?

**Pull request merged**